the-exterminators/error-annihilator
# SP-52 JDBC/PostgreSQL Dependency zum Projekt hinzufügen

With the Injecting of these dependencies developing and connecting to a database should now be possible on the Project

closes: SP-52 to Waiting for Approval

## Linked Issues
SP-52 JDBC/PostgreSQL Dependency zum Projekt hinzufügen

## PR Description

## Changes


## How to test

1. fetch Branch (follow PR guidelines in confluence https://se2-pg5-mci.atlassian.net/l/cp/1xjWiTXw)
2. open Branch in IDE (IntelliJ is highly recommended ;) )
3. check if methods and classes from h2, javax and jdbc/postgre can be used
4. If you haven't downloaded the Driver for jdbc yet, you might have to do this first

### Expected behaviour:
Developing on and connecting to Relational Databases should be possible now

## Checklist author


## Checklist reviewer/test-engineer



## Closes Issue
resolves SP-52
